_docs/concepts/security/authn-policy.md

```diff
@@ -0,0 +1,101 @@
+
+
+## Anatomy of the policy
+
+### Target selectors
+
+Defines rule to find service(s) on which policy should be applied. If no rule provided, the policy will be matched to all services in the namespace, so call namespace-level policy (as opposed to service-level policies which have non-empty selector rules). Istio (pilot) will pick the service-level policy if available, otherwise fallback to namespace-level policy. If neither is defined, it uses the default policy based on service mesh config and/or service annotation, which can only set mutual TLS setting (these are pre-0.7 mechanisms to config mutual TLS for Istio service mesh). See [testing Istio nutual TLS]({{home}}/docs/tasks/security/mutual-tls.html) and [per-service mutual TLS enablement]({{home}}/docs/tasks/security/per-service-mtls.html) for more details.
+
+Operators are responsible for avoiding conflicts, e.g create more than one service-level policy that match to the same service(s) (or more than one namespace-level policy on the same namespace).
+
+
+Example: rule to select product-page service (on any port), and reviews:9000.
+
+```
+ targets:
+ - name: product-page
+ - name: reviews
+   ports:
+   - number: 9000
+```
+
+### Peer authentication
+
+
+Defines authentication methods (and associated parameters) that are supported for peer authentication. It can list more than one methods; only one of them needs to be satisfied for the authentication to pass. However, in 0.7 releases, only mutual TLS is supported. Omitting this if peer authentication is not needed.
+
+
+Example of peer authentiation using mutual TLS:
+
+```
+  peers:
+  - mtls:
+```  
+
+Note: in 0.7, mtls settings doesn't require any parameter (hence `- mtls:` or `- mtls: null` declaration is sufficient). In future, it may carry arguments to provide different mTLS implementations.
+
+### Origin authentication
+
+Defines authentication methods (and associated parameters) that are supported for for origin authentication. Only JWT is supported for this, however, the policy can list multiple JWTs by diffrent issuers. Same as peers authentication, only one of the listed methods need to be satisfied for the authenticaiton to pass.
+
+```
+origins:
+- jwt:
+    issuer: "https://accounts.google.com"
+    jwksUri: "https://www.googleapis.com/oauth2/v3/certs"
+```
+
+### Principal binding
+
+Defines what is the principal from the authentiation. By default, it will use the value of peer's principal (and if peer authentication is not applied, it would be left unset). Policy writer can choose to overwrite it with USE_ORIGIN. In future, we will also support *conditional-binding* (e.g USE_PEER when peer is X, otherwise USE_ORIGIN)
```

Comment:
it will use the value of peer's -> of the peer's principal -> this will be the peer's principal